internal/interface/repository/implicitrepos/updates_through_queries.go

```diff
@@ -84,7 +81,7 @@ func (u *UpdatesRepoThroughQueries) GetSearchSubscriptionsUpdates(id model.UserI
 				lastArticleUpdateTimestamp = articleMeta.LastUpdateTimestamp
 			}
 		}
-		lastAccessTimestamp, err := u.historyRepo.GetSearchLastAccessTimestamp(id, query)
+		lastAccessTimestamp, err := u.searchSubsRepo.GetSearchLastAccessTimestamp(id, query)
 		if err != nil && err != domain.NeverAccessed {
```

Comment:
Make it more readable. Consider using early returns: https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
fixed